.travis.yml

```diff
@@ -56,3 +68,49 @@ script:
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then xvfb-run pytest --capture=no --cov=volumina; fi
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then pytest --capture=no --cov=volumina; fi
   - cd $TRAVIS_BUILD_DIR && git diff --name-only --diff-filter=AM master.. | grep ".*\.py" | xargs black --check
+
+
+jobs:
+  include:
+    - stage: deploy
+      os:
+        - linux
+      on:
+        branch: master
+        condition: $TRAVIS_OS_NAME = "linux"
+      name: "Deploy github pages"
+      install:
+        - sudo apt-get update
+        - export MINICONDA=https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
+        - echo $CONDA_ROOT
+        - |
+          if [[ ! -d ${CONDA_ROOT} ]]; then
+            echo "Installing Miniconda...";
+            wget $MINICONDA -O miniconda.sh &&
+            bash miniconda.sh -b -p ${CONDA_ROOT};
+          else
+            echo "Using cached Miniconda install";
+          fi
+        - echo ". $CONDA_ROOT/etc/profile.d/conda.sh" >> ~/.bashrc
+        - source ~/.bashrc
+        - conda config --set always_yes yes --set changeps1 no
+        - conda update -q conda
+        - |
+          if [[ ! -d deploy ]]; then
+            conda create -q -n deploy -c ilastik-forge -c conda-forge ilastik-dependencies-no-solvers sphinx;
+          else
+            conda install -y -n deploy -c ilastik-forge -c conda-forge ilastik-dependencies-no-solvers sphinx;
```

Comment:
Do we really need to install ilastik deps in this case? Can we install only sphinx?

Reply:
you spot every bit of lazyness I have in me. It didn't work without, so my quick fix was to download the whole thing. There are parts in `docs/conf.py` that require `lazyflow` for nicer display of string names or something... I'll investigate